.vscode/settings.json

```diff
@@ -1,7 +1,6 @@
 {
     "python.testing.unittestEnabled": true,
     "python.testing.pytestEnabled": false,
-    "python.pythonPath": "c:\\Work\\AzureIoT\\IoTEdge\\IoTEdgeHubDev\\py36venv\\Scripts\\python.exe",
```

Comment:
>pythonPath [](start = 12, length = 10)

can you debug the tests w/o the pythonPath explicitly specified?

Reply:
Yes - this change is prompted by vs code complaining about that line. Either way, it was environment specific :-)